tests/playbooks/test_minimal_ospdo.yaml

```diff
@@ -0,0 +1,42 @@
+- name: Prelude
+  hosts: local
+  gather_facts: false
+  module_defaults:
+    ansible.builtin.shell:
+      executable: /bin/bash
+  roles:
+    - prelude_local
+
+- name: Cleanup
+  hosts: local
+  gather_facts: false
+  module_defaults:
+    ansible.builtin.shell:
+      executable: /bin/bash
+  roles:
+    - pcp_cleanup
+
+
+- name: Adoption
+  hosts: local
+  gather_facts: false
+  module_defaults:
+    ansible.builtin.shell:
+      executable: /bin/bash
+  tasks:
+    - name: set ospdo_src var
```

Comment:
If all the ospdo specific task you have added in service roles are conditional, then the same test-minimal test suite could be used by defining conditional variable ospdo_src in vars.yaml.

Reply:
that's a good point , I'll change accordingly